HoloStorageAccessor/api_spec/openapi.yaml

```diff
@@ -89,8 +99,8 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Holograms'
-        400:
-          description: Unaccepted input
+        500:
+          $ref: "#/components/responses/InternalError"
 
     post:
```

Comment:
Just noticed: A successfull `POST` should return `202 Created`

Reply:
Do you mean 201?

Reply:
done